MovieEnthusiast.Api.Tests/MovieEnthusiast.Api.Tests.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
```

Comment:
Let's add the no-mercy configuration in this and other test project so that it follows our configuration in other projects
```
  <PropertyGroup>
    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
    <WarningLevel>9999</WarningLevel>
    <AnalysisLevel>preview</AnalysisLevel>
  </PropertyGroup>
```